frontend/src/hooks/routingPolicies/useGetRoutingPolicies.ts

```diff
@@ -0,0 +1,44 @@
+import {
+	getRoutingPolicies,
+	GetRoutingPoliciesResponse,
+} from 'api/routingPolicies/getRoutingPolicies';
+import { REACT_QUERY_KEY } from 'constants/reactQueryKeys';
+import { useMemo } from 'react';
+import { useQuery, UseQueryOptions, UseQueryResult } from 'react-query';
+import { ErrorResponse, SuccessResponse } from 'types/api';
```

Comment:
We should use `ErrorResponseV2` and `SuccessResponseV2`. These are new APIs and follow new error and success structure.